cmake/package_tooling.cmake

```diff
@@ -0,0 +1,113 @@
+# SPDX-License-Identifier: Apache-2.0
+#
+# Copyright (c) 2021, Nordic Semiconductor ASA
+
+# The Zephyr package tooling script provides a generic script mode interface
```

Comment:
Thanks for the proposal.
I'm not completely fond of `package_tooling.cmake`, and maybe `run_cmake_module.cmake` is better.
Downside with that name is that it can give the impression that it's possible to run any CMake module, when in fact it only allows to run Zephyr cmake modules (and it does so through the `find_package(Zephyr)` call).

Before changing, I would like to see of other good proposals arises.

Reply:
> Maybe run_cmake_module.cmake would be a better name for this file?

How about calling it `breeze.cmake` ?

Then `west build` can do a full Zephyr or just a light breeze in future :wink:

Reply:
> How about calling it breeze.cmake ?

Discussed offline with @carlescufi - both of us think that the name should make it obvious what this file does instead, so `breeze` doesn't fit the bill

Reply:
How about `package_helper`?

Reply:
`package_helper` is fine with me.

Reply:
Care to rename this to package_helper, then? I am happy to approve after that.

Reply:
done.

Was just being a bit passive on this part, as I'm anyway waiting for #41301 to go in first.